config/complaint-form-email-converter/templates/receiverEmail.js

```diff
@@ -1,7 +1,6 @@
-import { format } from 'date-fns';
-import { formatInTimeZone } from 'date-fns-tz';
 import envvar from 'env-var';
 import * as uti from './utils';
+import { formatDate } from './utils';
```

Comment:
Did it like this because I really don't like the `uti.` prefix that is used for the other utils in the model 😄. I think named exports are nicer to read. I can change it if people have a strong preference, but I would prefer if we go the other way instead (but didn't want to make that part of this PR).

Reply:
I agree as well; the named import is nicer and clearer to read.